clients/ui/bff/internal/api/errors.go

```diff
@@ -91,18 +91,19 @@ func (app *App) methodNotAllowedResponse(w http.ResponseWriter, r *http.Request)
 	app.errorResponse(w, r, httpError)
 }
 
-func (app *App) failedValidationResponse(w http.ResponseWriter, r *http.Request, errors map[string]string) {
-
-	message, err := json.Marshal(errors)
-	if err != nil {
-		message = []byte("{}")
-	}
-	httpError := &integrations.HTTPError{
-		StatusCode: http.StatusUnprocessableEntity,
-		ErrorResponse: integrations.ErrorResponse{
-			Code:    strconv.Itoa(http.StatusUnprocessableEntity),
-			Message: string(message),
-		},
-	}
-	app.errorResponse(w, r, httpError)
-}
+// TODO: Uncomment when we implement proper error handling
```

Comment:
Why did we have to comment on this? To please the linter?

Reply:
Yeah it was an unused function so the linter was upset. We've been doing similar commenting out in the frontend so I figured to do the same here

Reply:
Decided to change this so that we use `//nolint`